### ISSUE TITLE
Screen Time Reminder

### ISSUE THREAD

**Issue by RahiAkherajot** [UriahShaulMandel/BaldPhone#224]
Using smartphone is a fun
But there are so many harmful effects of using smartphone for a long time
We forget the time passing while using smartphone
Therefore a feature is needed which reminds the user about the time invested on the smartphone

**Comment by RahiAkherajot** [comment]
This is a simple feature which monitors time after phone has been unlocked
It will sound a beep after a customisable given time
It beeps twice after the twice of the given time passed

**Comment by RahiAkherajot** [comment]
More features in a single app may increase the app size but this will make the best use of any smartphone
Using two different apps for two features will use more space and data
BaldPhone being committed to privacy will attract more users.

**Comment by aha999** [comment]
I believe the targeted users aren't using smartphones for extended periods of time anyways. The function is present in the battery info

**Comment by UriahShaulMandel** [comment]
I agree with @aha999 , and won't do it myself in the next fee months, but if anyone wants to open a pr, you're all welcome to do so :)

**Comment by RahiAkherajot** [comment]
This app should run on this principle 'Prevention is  better than Cure'
By the way I think BaldPhone isn't much popular among the needed and every launcher announces itself 'The Simplest'
Adding exclusive features will make it popular and the targeted users will come in contact with this app.
This feature may be disabled by default

**Comment by UriahShaulMandel** [comment]
I understand what you say :), and reopening this issue,
also, please remember: 

> BaldPhone is a platform which replaces the phone's interface to a big, simple and friendly one. 
> It is built for elderly people, people with motoric problems and people who need a visual aid.

I have limited time and resources and would like to prioritize tasks based on the base users' needs. 
I won't do it myself in the close months, just in the name of prioritization, other issues are more urgent IMO. If anyone wants to PR, feel free!
Thanks for the cool idea!
